usafacts/tests/conftest.py

```diff
@@ -11,7 +11,9 @@
 @pytest.fixture(scope="session")
 def run_as_module():
     # Clean receiving directory
-    for fname in listdir("receiving"):
-        remove(join("receiving", fname))
+    for fname in listdir("../receiving"):
+        if fname[0] == ".":
+            continue
+        remove(join("../receiving", fname))
```

Comment:
I think this can be made into a one liner with

 `[remove(fname) for fname in glob.glob('receiving/*')]`

Reply:
oops realized this is part of #314